docs/usage.md

```diff
@@ -19,6 +19,8 @@
   * [`--barcode_kit`](#--barcode_kit)
   * [`--guppy_config`](#--guppy_config)
   * [`--guppy_gpu`](#--guppy_gpu)
+  * [`--guppy_gpu_runners_per_device`](#--guppy_gpu_runners_per_device)
```

Comment:
This has gone back to the old names?

Reply:
🤦‍♂ sorry about that. I think they were never changed from the first suggestion. Should be fixed now.